specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/SourceControls.json

```diff
@@ -171,6 +171,13 @@
               "$ref": "#/definitions/SourceControl"
             }
           },
+          {
+            "name": "x-ms-github-auxiliary",
```

Comment:
>x-ms-github-auxiliary [](http://example.com/codeflow?start=21&length=21)

who sends this header ? and curious on what/how is it used ?

Reply:
customer needs to provide the header to pass in the Github token for API backend implementation requirement